### PR TITLE
Add County Dropdown

### PR DESCRIPTION
**Pull request by pjaudiomv**
@paulnagle  ref #7

### CODE REVIEW

File: js/bmlt_tabs.js [hunk 1]
```diff
@@ -203,6 +232,7 @@ jQuery(document).ready(function($) {
 		if ( $( "#e4" ).length ) { $("#e4").select2("val", null); }
 		if ( $( "#e5" ).length ) { $("#e5").select2("val", null); }
 		if ( $( "#e6" ).length ) { $("#e6").select2("val", null); }
+		if ( $( "#e7" ).length ) { $("#e6").select2("val", null); }
```
Comment (radius314) [inline js/bmlt_tabs.js]: should be `$("#e7").select2("val", null)`

File: js/bmlt_tabs.js [hunk 2]
```diff
@@ -181,12 +194,28 @@ jQuery(document).ready(function($) {
...
+			return;
+        });
+    });
     $("#day").on('click', function() {
 		if ( $( "#e2" ).length ) { $("#e2").select2("val", null); }
 		if ( $( "#e3" ).length ) { $("#e3").select2("val", null); }
 		if ( $( "#e4" ).length ) { $("#e4").select2("val", null); }
 		if ( $( "#e5" ).length ) { $("#e5").select2("val", null); }
 		if ( $( "#e6" ).length ) { $("#e6").select2("val", null); }
+		if ( $( "#e7" ).length ) { $("#e6").select2("val", null); }
```
Comment (radius314) [inline js/bmlt_tabs.js]: should be `$("#e7").select2("val", null)`

File: js/bmlt_tabs.js [hunk 3]
```diff
@@ -172,6 +184,7 @@ jQuery(document).ready(function($) {
 		if ( $( "#e3" ).length ) { $("#e3").select2("val", null); }
 		if ( $( "#e4" ).length ) { $("#e4").select2("val", null); }
 		if ( $( "#e5" ).length ) { $("#e5").select2("val", null); }
+		if ( $( "#e7" ).length ) { $("#e5").select2("val", null); }
```
Comment (radius314) [inline js/bmlt_tabs.js]: should be `$("#e7").select2("val", null)`

File: js/bmlt_tabs.js [hunk 4]
```diff
@@ -158,6 +169,7 @@ jQuery(document).ready(function($) {
 		if ( $( "#e3" ).length ) { $("#e3").select2("val", null); }
 		if ( $( "#e4" ).length ) { $("#e4").select2("val", null); }
 		if ( $( "#e6" ).length ) { $("#e6").select2("val", null); }
+		if ( $( "#e7" ).length ) { $("#e6").select2("val", null); }
```
Comment (radius314) [inline js/bmlt_tabs.js]: should be `$("#e7").select2("val", null)`

File: js/bmlt_tabs.js [hunk 5]
```diff
@@ -144,6 +154,7 @@ jQuery(document).ready(function($) {
 		if ( $( "#e3" ).length ) { $("#e3").select2("val", null); }
 		if ( $( "#e5" ).length ) { $("#e5").select2("val", null); }
 		if ( $( "#e6" ).length ) { $("#e6").select2("val", null); }
+		if ( $( "#e7" ).length ) { $("#e6").select2("val", null); }
```
Comment (radius314) [inline js/bmlt_tabs.js]: should be `$("#e7").select2("val", null)`

File: js/bmlt_tabs.js [hunk 6]
```diff
@@ -130,6 +139,7 @@ jQuery(document).ready(function($) {
 		if ( $( "#e4" ).length ) { $("#e4").select2("val", null); }
 		if ( $( "#e5" ).length ) { $("#e5").select2("val", null); }
 		if ( $( "#e6" ).length ) { $("#e6").select2("val", null); }
+		if ( $( "#e7" ).length ) { $("#e6").select2("val", null); }
```
Comment (radius314) [inline js/bmlt_tabs.js]: should be `$("#e7").select2("val", null)`

**Comment by pjaudiomv** [comment]
this is tested and is whats on here https://charlestonna.org/ignoreform/

**Comment by radius314** [comment]
fyi, you can test this locally as you did with bread.  i don't necessarily expect to see proof, but if there are obvious code issues, it's a smell that testing was not done right before the pull request was made.

**Pull request merged**